### DISCUSSION TITLE
Is it possible to export Pag animation to video in web?

### DISCUSSION THREAD

**Discussion by zdarovich** [Tencent/libpag#2567] [Q&A] [answered]
How can i export pag animation to video in web?

**Comment by kevingpqi123** [comment, marked as answer]
In theory, rendering and exporting videos on the web platform is possible. However, rendering and encoding are computationally intensive operations, and for performance reasons, it is generally not recommended to do so directly on the client side. A common practice is to perform effect rendering and exporting video on the server side, while the web platform is used for simple interactions and visual effects. This approach is commonly used in scenarios such as Tencent Video advertisements and WeChat game reports.

Since pag is an animation rendering library, the task of rendering and exporting videos would need to be developed by the business or application developers, as it falls outside the scope of the library's functionality.
